apps/vendors/tests/test_api.py

```
@@ -0,0 +1,39 @@
+"""API tests for vendors."""
+import pytest
+from rest_framework.reverse import reverse
+
+
+@pytest.mark.django_db
+def test_vendor_list_returns_verified_entries(api_client, user_factory, vendor_factory):
+    """Customers should see only verified vendors."""
+
+    visible_vendor = vendor_factory(name='Visible Clinic', is_verified=True, is_active=True)
```

Ruff (0.14.0)

Local variable `visible_vendor` is assigned to but never used

Remove assignment to unused variable `visible_vendor`

(F841)